pkg/hintrunner/zero/zerohint_usort_test.go

```diff
@@ -13,9 +13,10 @@ func TestZeroHintUsort(t *testing.T) {
 		"UsortEnterScope": {
 			{
 				ctxInit: func(ctx *hinter.HintRunnerContext) {
-					ctx.ScopeManager.EnterScope(map[string]any{
-						"__usort_max_size": feltUint64(1),
-					})
+					err := ctx.ScopeManager.AssignVariable("__usort_max_size", feltUint64(1))
+					if err != nil {
+						t.Fatal(err)
```

Comment:
In most of the unit tests that were modified in this PR, `panic(err)` was used instead. I see you've changed that in this file. I'd go with `t.Fatal(err)` as well, as this allows other tests to continue running instead of stoping all the execution.